analysis/tests/src/expected/Completion.res.txt

```diff
@@ -918,13 +918,13 @@ Path Js.n
     "label": "nullable",
     "kind": 22,
     "tags": [],
-    "detail": "type nullable<+'a>",
-    "documentation": {"kind": "markdown", "value": "\n  A value of this type can be undefined, null or 'a. This type is equivalent to Js.Null_undefined.t.\n"}
+    "detail": "type nullable<'a> = Value('a) | Null | Undefined",
```

Comment:
@zth we lost the `+` here.

Reply:
Right, that's... weird. Don't think we changed the `+` in the implementation...

Reply:
Looks like this is the intended printing, inherited from OCaml, which only prints the `+` when the type is abstract:

```ocaml
# type 'a t1;;
type 'a t1
# type +'a t2;;
type +'a t2
# type +'a t2 = | A;;
type 'a t2 = A
```

Reply:
Interesting! That's good to know 😀